src/components/ButtonPanel.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from './Button';
+import styles from '../css/modules/ButtonPanel.module.css';
+
+const buttonsLabel = [
+  { group: ['AC', '+/-', '%', '÷'], id: 'group-one' },
+  { group: ['7', '8', '9', 'X'], id: 'group-two' },
+  { group: ['4', '5', '6', '-'], id: 'group-three' },
+  { group: ['1', '2', '3', '+'], id: 'group-four' },
+  { group: ['0', '.', '='], id: 'group-five' },
+];
+
+class ButtonPanel extends React.Component {
+  renderButtonGroup({ group, id, key }) {
+    const { clickHandler } = this.props;
+    const buttonGroups = group.map((label, index) => (
+      <Button
+        key={label}
+        color={index < group.length - 1 ? 'lightgrey' : 'orange'}
+        name={label}
+        isWide={label === '0'}
+        onClick={() => clickHandler(label)}
+      />
+    ));
```

Comment:
Please pass `handleClick` as `clickHandler` props to each `Button` 
We can agree that it can be confusing to pass a prop named "onClick"  😊

Reply:
I don't think this should be a **required** changes.

It is not confusing for me. `onClick` is a valid prop name I think. Please see this link (https://reactjs.org/tutorial/tutorial.html) and the screenshot below.

![image](https://user-images.githubusercontent.com/36057474/69125482-2fbd3780-0aa6-11ea-908b-d867c387a55e.png)